src/Cell.js

```diff
@@ -348,7 +386,7 @@ const Cell = React.createClass({
     let dragHandle = (!this.isActive() && this.canEdit()) ? <div className="drag-handle" draggable="true" onDoubleClick={this.onDragHandleDoubleClick}><span style={{display: 'none'}}></span></div> : null;
 
     return (
-      <div {...this.props} className={className} style={style} onClick={this.onCellClick} onDoubleClick={this.onCellDoubleClick} onDragOver={this.onDragOver}>
+      <div {...this.props} className={className} style={style} onClick={this.columnEventDispatcher} onDoubleClick={this.columnEventDispatcher} onDragOver={this.columnEventDispatcher}>
```

Comment:
If we use {...column.events} spread syntax into the div we can dynamically bind each column event rather than hardcode them as above. Although, we should leave onClick and onDoubleClick, and make sure default functionality of those two events is not overwritten